synfig-studio/src/gui/dialogs/dialog_pluginmanager.cpp

```diff
@@ -0,0 +1,126 @@
+/* === S Y N F I G ========================================================= */
+/*!	\file dialogs/dialog_pluginmanager.h
+**	\brief Implementation of dialog to install, uninstall and manage plugins
+**
+**	$Id$
+**
+**	\legal
+**	Copyright (c) 2002-2005 Robert B. Quattlebaum Jr., Adrian Bentley
+**
+**	This package is free software; you can redistribute it and/or
+**	modify it under the terms of the GNU General Public License as
+**	published by the Free Software Foundation; either version 2 of
+**	the License, or (at your option) any later version.
+**
+**	This package is distributed in the hope that it will be useful,
+**	but WITHOUT ANY WARRANTY; without even the implied warranty of
+**	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+**	General Public License for more details.
+**	\endlegal
+*/
+/* ========================================================================= */
```

Comment:
Please follow the current template for .cpp

Reply:
Where should I find it ?

Reply:
I just found out where did you get this old template.
synfig-studio/src/gui/dialogs/canvasresize.h / synfig-studio/src/gui/dialogs/canvasresize.cpp

Any other file uses the correct one :)

https://github.com/synfig/synfig/blob/master/synfig-core/src/template.h

https://github.com/synfig/synfig/blob/master/synfig-core/src/template.cpp